### DISCUSSION TITLE
Underline, italic, ... textStyles per theme

### DISCUSSION THREAD

**Discussion by jderboven** [chakra-ui/panda#2402] [Q&A]
Hello all,
I'd like to know if there is any mean to tokenize those properties : textTransform with underline, italic also, ... I haven't found anything but that would be really great, I have 2 themes on my app and many more coming but between them, the difference is some text might be underlined or not ... Could you help ?

**Comment by astahmer** [comment]
it sounds like this could be defined in a recipe, based on variants
